.github/workflows/windows.yaml

```diff
@@ -0,0 +1,98 @@
+name: Windows build
+
+on:
+  push:
+    branches:
+      - v2
+  pull_request:
+    branches:
+      - v2
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build integration for
+    strategy:
+      fail-fast: false
+      matrix:
+        windows:
+          # Here we specify the GH runner where the image will be built.
+          # Tag must exist in both https://hub.docker.com/_/microsoft-windows-servercore and
+          # https://hub.docker.com/_/microsoft-windows-nanoserver, and must be matched with the runner.
+          # In theory, newer versions could build old images using --isolation=hyperv, but unfortunately hyperv is not
+          # enabled in GHA.
+          - runner: windows-2019
+            tag: 1809
+          - runner: windows-2022
+            tag: ltsc2022
+    runs-on: ${{ matrix.windows.runner }}
+    env:
+      AGENT_VERSION: 1.20.7
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
+      - name: Get windows build
+        id: osinfo
+        shell: powershell
+        run: |
+          Get-ComputerInfo
```

Comment:
is this needed?

Reply:
Previous iterations of this script used the output of this step to determine the windows version programatically. Now it is defined by hand on the matrix, but I thought the command was still useful as it might help in debugging. It tells you the details of the windows version, including build number etc.